src/main/java/org/team1540/bobafett/RobotContainer.java

```diff
@@ -0,0 +1,75 @@
+
+package org.team1540.bobafett;
+
+import com.ctre.phoenix.sensors.Pigeon2;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import org.photonvision.PhotonCamera;
+import org.team1540.bobafett.commands.claw.*;
+import org.team1540.bobafett.commands.drivetrain.Drivetrain;
+import org.team1540.bobafett.commands.drivetrain.TankDrive;
+import org.team1540.bobafett.commands.elevator.*;
+import org.team1540.bobafett.commands.drivetrain.AprilTagPIDTurn;
+import org.team1540.bobafett.utils.ChickenPhotonCamera;
+
+import java.util.function.BiConsumer;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and button mappings) should be declared here.
+ */
+public class RobotContainer {
+    // The robot's subsystems and commands are defined here...
+    private final Pigeon2 pigeon = new Pigeon2(0);
+    private final Drivetrain drivetrain = new Drivetrain(pigeon);
+    private final ChickenPhotonCamera camera = new ChickenPhotonCamera(Constants.VisionConstants.CAMERA_NAME);
+    private final XboxController controller = new XboxController(0);
+    private final Claw claw = new Claw();
+    private final Elevator elevator = new Elevator();
+
+    /** The container for the robot. Contains subsystems, OI devices, and commands. */
+    public RobotContainer() {
+        // Configure the button bindings
+        configureButtonBindings();
+        drivetrain.setDefaultCommand(new TankDrive(drivetrain, controller));
+    }
+
+    /**
+     * Use this method to define your button->command mappings. Buttons can be created by
+     * instantiating a {@link GenericHID} or one of its subclasses ({@link
+     * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
+     * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
+     */
+    private void configureButtonBindings() {
+        new JoystickButton(controller, XboxController.Button.kX.value)
+                .whenPressed(new AprilTagPIDTurn(drivetrain, camera, Constants.VisionConstants.APRIL_TAG_ID));
+        new JoystickButton(controller, XboxController.Button.kB.value)
+                .whileActiveContinuous(new CloseClaw(claw));
+        new JoystickButton(controller, XboxController.Button.kY.value)
+                .whenPressed(new MoveToTop(elevator));
+        new JoystickButton(controller, XboxController.Button.kLeftBumper.value)
+                .whenPressed(new InstantCommand(elevator::setPosition1)); // TODO adjust setpoints
```

Comment:
Writing specific methods for this works, but if you want to avoid needing two extra functions for this, you can use a lambda instead of a method reference 
```java
() -> elevator.setPidReference(60)
```